scripts/bootstrap/compile.sh

```diff
@@ -16,8 +16,14 @@
 
 # Script for building bazel from scratch without bazel
 
-PROTO_FILES=$(ls src/main/protobuf/*.proto src/main/java/com/google/devtools/build/lib/buildeventstream/proto/*.proto src/main/java/com/google/devtools/build/skyframe/*.proto src/main/java/com/google/devtools/build/lib/skyframe/proto/*.proto)
-LIBRARY_JARS=$(find derived/jars third_party -name '*.jar' | grep -Fv JavaBuilder | grep -Fv third_party/guava | grep -Fv third_party/guava | grep -ve 'third_party/grpc/grpc.*jar' | tr "\n" " ")
+if [ -d derived/jars ]; then
+  PROTOBUF_JARS=derived/jars
+else
+  PROTOBUF_JAVA_VERSION=3.11.4
+  PROTOBUF_JARS="/usr/share/java/protobuf-$PROTOBUF_JAVA_VERSION.jar /usr/share/java/protobuf-java-util-$PROTOBUF_JAVA_VERSION.jar"
```

Comment:
Is protobuf-java's behavior likely to change enough from one release to the next that the versioning is necessary? Note that this will need to be updated whenever the local system's version of these packages changes, and this is likely to not be in sync with Bazel updates/releases. I'd recommend using the non-versioned name as long as version changes are unlikely to break the functionality needed by Bazel. If protobuf changes **are** likely to break Bazel then default assignment ( :=) should be used instead of normal assignment (=) to allow users to easily override this value.

Reply:
I'm not sure if they have improved, but in the v2 release series we saw breaking changes on minor releases.

Reply:
Maybe we can specify what protobuf version or range or version Bazel could depend on in the Debian package? So that here, we can just use the non-versioned name?